pact3.0/01-coin-gov.repl

```diff
@@ -0,0 +1,93 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Example of using module governance capability
+
+
+; --------------------------------------------------------------------------
+; Install account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module"
+
+  (defcap ACCT-GOVERNANCE ()
+    ;;; module can only be upgraded if admin-keyset is satisfied.
+    ;;; i.e. If transaction signed by both "product-dept" and "dev-dept"
+    (enforce-guard (read-keyset "admin-keyset")))
+
+  (defschema account
+    balance:integer
+    ks:keyset))
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Upgrades schema by triggering module's governance capability.
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-keys ["product-dept" "dev-dept"])
```

Comment:
add comment on line before "; simulate multi-signature of product and dev department"